Library/Homebrew/formula.rb

```diff
@@ -1455,16 +1455,6 @@ def system(cmd, *args)
       $stdout.flush
 
       unless $?.success?
-        log_lines = ENV["HOMEBREW_FAIL_LOG_LINES"]
-        log_lines ||= "15"
-
-        log.flush
-        if !verbose || verbose_using_dots
-          puts "Last #{log_lines} lines from #{logfn}:"
-          Kernel.system "/usr/bin/tail", "-n", log_lines, logfn
-        end
-        log.puts
-
```

Comment:
`HOMEBREW_FAIL_LOG_LINES` is set to 150 in the `test-bot`, so it is not always 15. We need to preserve this functionality for the bots. Personally, I would hope we can retain this also for the non-bot case. How about guarding the lines you deleted here with the following code instead, so that normal users won't see the excerpt?

```ruby
if ARGV.homebrew_developer? || ENV["HOMEBREW_FAIL_LOG_LINES"]
```